src/site/stages/build/process-cms-exports/transformers/node-health_care_local_health_service.js

```diff
@@ -4,13 +4,14 @@ const transform = entity => ({
   entity: {
     entityType: 'node',
     entityBundle: 'health_care_local_health_service',
+    title: getDrupalValue(entity.title),
```

Comment:
I added `title` to be able to test and ensure that `getDrupalValue` will work on both cases